templates/servicemonitor.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and (.Values.metrics.enabled) (.Values.metrics.serviceMonitor.enabled)}}
+kind: ServiceMonitor
+apiVersion: monitoring.coreos.com/v1
+metadata:
+  labels:
+    {{- include "qdrant.labels" . | nindent 4 }}
+  name: {{ include "qdrant.fullname" . }}
+spec:
+  endpoints:
+    - honorLabels: true
+      interval: {{ .Values.metrics.serviceMonitor.scrapeInterval }}
+      path: {{ .Values.metrics.serviceMonitor.targetPath }}
+      port: {{ .Values.metrics.serviceMonitor.targetPort }}
+      scheme: http
+      scrapeTimeout: {{ .Values.metrics.serviceMonitor.scrapeTimeout }}
+  selector:
+    matchLabels:
+      {{- include "qdrant.selectorLabels" . | nindent 6 }}
```

Comment:
```suggestion
      {{- include "qdrant.labels" . | nindent 6 }}
```

Reply:
Since this is a service monitor, I believe it should matchLabels of the qdrant service labels not the qdrant pods labels

Reply:
Update made, it was a slight oversight